ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/manager/RespondingEthPeer.java

```diff
@@ -217,12 +218,21 @@ public boolean respond(final Responder responder) {
   }
 
   private void respondToMessage(final Responder responder, final OutgoingMessage msg) {
-    final Optional<MessageData> maybeResponse = responder.respond(msg.capability, msg.messageData);
+    MessageData messageData = msg.messageData;
```

Comment:
I wouldn't mind an if / else here to make it clearer

Reply:
done, this code will be removed soon anyway :)